aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -102,6 +109,13 @@ module aptos_framework::account {
     const PROLOGUE_ESEQUENCE_NUMBER_TOO_BIG: u64 = 1009;
     const PROLOGUE_ESECONDARY_KEYS_ADDRESSES_COUNT_MISMATCH: u64 = 1010;
 
+    #[test_only]
+    const OFFER_ROTATION_CAPABILITY_SIGNATURE: vector<u8> = x"68f7d09ef7a9d8d7450d600b10231e6512610f919a63bd71bea1c907f7e101ed333bff360eeda97a8637a53fd622d597c03a0d6fd1315c6fa23719983ff7de0c";
+    #[test_only]
+    const OFFER_ROTATION_CAPABILITY_INVALID_SIGNATURE: vector<u8> = x"78f7d09ef7a9d8d7450d600b10231e6512610f919a63bd71bea1c907f7e101ed333bff360eeda97a8637a53fd622d597c03a0d6fd1315c6fa23719983ff7de0c";
+    #[test_only]
+    const OFFER_ROTATION_CAPABILITY_PUBLIC_KEY: vector<u8> = x"f66bf0ce5ceb582b93d6780820c2025b9967aedaa259bdbb9f3d0297eced0e18";
+
```

Comment:
maybe move below to the tests themselves.

`verify_authentication_key_matches_ed25519_public_key`